dali/pipeline/workspace/support_workspace.h

```diff
@@ -54,6 +54,15 @@ class DLL_PUBLIC SupportWorkspace : public WorkspaceBase<SupportInputType, Suppo
    */
   template <typename Backend>
   DLL_PUBLIC Tensor<Backend>& Output(int idx);
+
+  bool has_stream() const override {
+    return false;
+  }
+
+ private:
+  cudaStream_t stream_impl() const override {
+    return nullptr;
```

Comment:
According to https://docs.nvidia.com/cuda/cuda-runtime-api/stream-sync-behavior.html (`The default stream, used when 0 is passed as a cudaStream_t`) use `0` instead of nullptr if you want to use the default stream.